evaluator/evaluator.py

```diff
@@ -495,12 +504,13 @@ def process_message(message):
                     return
                 if msg_dict['type'] == 'vmaas_updates':
                     LOGGER.info("Received message type: %s", msg_dict['type'])
+                    LOGGER.info("VMaaS metadata changed, need to re-evaluate all systems")
                     self._sync_cve_md()
-                    cur.execute("select inventory_id, vmaas_json from system_platform")
+                    cur.execute("select inventory_id from system_platform")
                     # reevaluate updates for every system in the DB
-                    for system_platform in cur.fetchall():
-                        self.evaluate_vmaas(system_platform)
-                elif msg_dict['type'] == 'upload_new_file':
+                    for inventory_id, in cur.fetchall():
```

Comment:
extraneous comma?

Reply:
no, this is a way how to extract single attribute from tuple, otherwise I'd need to get it like `inventory_id[0]`

Reply:
Ah, I see - kk, I assume this works then :)